lua/wire/server/wirelib.lua

```diff
@@ -1537,9 +1537,20 @@ if not WireLib.PatchedDuplicator then
 end
 
 function WireLib.SoundExists(path)
-	path = string.GetNormalizedFilepath(string.gsub(string.sub(path, 1, 260), '["?]', ''))
+	-- Limit length and remove invalid chars
+	if #path>260 then path = string.sub(path, 1, 260) end
```

Comment:
Is there any reason for having this? iirc this was only put into place as a band aid fix for sounds too long for source to play, but since now there's file path checking isn't it unnecessary?